backend_jobs/ingestion_pipeline/setup.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
file would benefit from some doc (what is this for?)

Reply:
added doc